### PR TITLE
Pacemaker

### PR DESCRIPTION
**Pull request by Sereni**
Position target score and remove target graph
Screenshot

- Make the score visible with and without NPS graph
- Remove target graph code as it will not fit onto the screen — might revisit with a very narrow bar on the edge, later
- Remove the option to select target score or data visualizations as neither will be properly visible

### CODE REVIEW

**Comment by Sereni** [comment]
Please ignore the first commit — it's equivalent to the PR from `target`, and I will learn to git eventually :<

File: BGAnimations/ScreenGameplay underlay/PerPlayer/Score.lua [hunk 1]
```diff
@@ -27,9 +26,9 @@ return LoadFont("_wendy monospace numbers")..{
 	Name=pn.."Score",
 	InitCommand=function(self)
 		self:valign(1):halign(1)
-                self:zoom(Positions.ScreenGameplay.ScoreZoom())
```
Comment (Fieoner) [inline BGAnimations/ScreenGameplay underlay/PerPlayer/Score.lua:30]: Wouldn't it make sense to use Positions.whatever here?

File: BGAnimations/ScreenGameplay underlay/PerPlayer/DifficultyMeter.lua [hunk 2]
```diff
@@ -2,7 +2,7 @@ local player = ...
 
 return Def.ActorFrame{
 	InitCommand=function(self)
-		self:xy( _screen.w-16, 16 )
+		self:xy( _screen.w-17, 16 )
```
Comment (Fieoner) [inline BGAnimations/ScreenGameplay underlay/PerPlayer/DifficultyMeter.lua:5]: This is part of pr #39 too D:

**Comment by Fieoner** [comment]
image
Is that position right? It looks good with background disabled but a bit weird with bg on

File: BGAnimations/ScreenGameplay underlay/PerPlayer/Score.lua (hunk 1 above)
Comment (Sereni) [reply BGAnimations/ScreenGameplay underlay/PerPlayer/Score.lua:30]: Nah — it was an attempt to standardize where things are on the screen, but it's never reused, and pretty useless when we apply zoom on top.

**Comment by Sereni** [comment]
Nice catch, moved it up a bit.

**Pull request merged**